ext/JACCAMDGPU/JACCAMDGPU.jl

```diff
@@ -2,8 +2,6 @@ module JACCAMDGPU
 
 using JACC, AMDGPU
 
-include("array.jl")
```

Comment:
The ones in other back ends might need to be removed

Reply:
I am not sure I understand this comment; I removed the `array.jl` from CUDA backend. I do see that there are some instances of `OneAPI.zeros` in that backend however I do not currently have access to any intel GPU cores to verify that code

Reply:
Oh thanks I missed that in JACCCUDA and I don't see these functions in JACCOneApi